d/dask/dask_ubi_9.3.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package          : dask
+# Version          : 2.20.0
+# Source repo      : https://github.com/dask/dask.git
+# Tested on        : UBI:9.3
+# Language         : Python
+# Travis-Check     : True
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Rakshith B R <rakshith.r5@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ---------------------------------------------------------------------------
+
+# Variables
+PACKAGE_NAME=dask
+PACKAGE_VERSION=${1:-2.20.0}  # Default version set to 2.20.0
+PACKAGE_URL=https://github.com/dask/dask.git
+
+# Install necessary system dependencies
+yum install -y git gcc gcc-c++ make wget python3-devel python3-pip libyaml-devel
+
+# Upgrade pip and install setuptools, wheel
+pip install --upgrade pip setuptools wheel
+
+# Clone the repository
+git clone $PACKAGE_URL
+cd $PACKAGE_NAME  
+git checkout $PACKAGE_VERSION  
+
+# Install Dask
+if ! pip install .; then
+    echo "------------------$PACKAGE_NAME:Install_fails-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail | Install_Fails"
+    exit 1
+fi
+
+# Build and install the package
+if ! python3 setup.py install; then
+    echo "------------------$PACKAGE_NAME: Install failed-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail | Install_Failed"
+    exit 1
+fi
+
+# Run tests
+if ! pytest --disable-warnings; then
```

Comment:
I don't see else block where it says "Both_Install_and_Test_Success" and exist 0